client/src/components/common/Cards/index.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import './style.css';
+
+const Card = (props) => {
+  const { values } = props;
+  return (
+    <ul className="box">
+      {values.map((element) => {
+        const { value, id } = element;
+        return (
+          <li index={id} className="box__card">{value}</li>
```

Comment:
use keys when creating lists from arrays.
https://reactjs.org/docs/lists-and-keys.html

for more explanation checkout this link:
https://stackoverflow.com/questions/42801343/what-is-the-significance-of-keys-in-reactjs/42801409

and this link:
http://jsfiddle.net/frosas/S4Dju/